pkg/pool-weighted/hardhat.config.ts

```diff
@@ -18,7 +18,18 @@ export default {
     },
   },
   solidity: {
-    compilers: hardhatBaseConfig.compilers,
+    // compilers: hardhatBaseConfig.compilers,
```

Comment:
This is probably something we should leave as was to avoid introducing issues to the monorepo